Sources/Common/Core/Math/index.js

```diff
@@ -1873,6 +1873,43 @@ export function areBoundsInitialized(bounds) {
   return !(bounds[1] - bounds[0] < 0.0);
 }
 
+export function computeBoundsCenter(bounds) {
+  return [
+    (bounds[0] + bounds[1]) / 2,
+    (bounds[2] + bounds[3]) / 2,
+    (bounds[4] + bounds[5]) / 2,
+  ];
+}
+
+export function computeBoundsScale3(bounds) {
+  const center = computeBoundsCenter(bounds);
+
+  return [
+    Math.max(bounds[0], bounds[1]) - center[0],
+    Math.max(bounds[2], bounds[3]) - center[1],
+    Math.max(bounds[4], bounds[5]) - center[2],
+  ];
+}
+
+export function computeBoundsPoints(point1, point2, bounds) {
```

Comment:
rename to `computeBoundsCornerPoints` and add a comment to explain what corner points.